### PR TITLE
Feature/hero

### PR DESCRIPTION
**Pull request by breakdancingcat**
* Removed 'view more' 
* Exchanged excerpt and view more with a description
* Added a cascading option for 'white links'; Didn't think this case warranted a hero component considering it could have multiple applications
* Removed the blur on text shadow because it was overlapping the underline and creating a weird appearance

### CODE REVIEW

**Comment by nickdenardis** [comment]
Before and after screenshots?

**Comment by breakdancingcat** [comment]
### Before
base-hero-before

### After
base-hero-after 
base-hero-after-small

**Comment by nickdenardis** [comment]
If we are going to trasition to the description field in the hero, we should limit the tags to only `<p>` and `<a>` tags, otherwise things could get dangerous. 

Because this updates the contents on the homepage thinking when this is released:

1. We need to update the initial promotion groups created when a site is created
2. Update the instructions on what each field does in the CMS hero promo group

**Comment by breakdancingcat** [comment]
@nickdenardis I chose to strip all tags except <a> because we shouldn't have multiple lines there anyway! :)

File: resources/views/components/hero.blade.php [hunk 1]
```diff
@@ -10,9 +10,8 @@
...
                     <div class="row">
                         <div class="mx-4">
-                            <h1 id="hero-image-{{ $loop->iteration }}" class="leading-tight text-2xl @if(in_array($page['controller'], config('base.hero_full_controllers')))xl:text-5xl @else xl:text-3xl @endif">{{ $image['title'] }}</h1>
-                            @if(!empty($image['excerpt']))<p class="inline pr-2 md:text-lg">{{ $image['excerpt'] }}</p>@endif
-                            @if(!empty($image['link']))<a href="{{ $image['link'] }}" class="underline font-bold hover:no-underline md:text-white md:text-lg">{{ config('base.hero_text_more') }}</a>@endif
+                            <h1 id="hero-image-{{ $loop->iteration }}" class="leading-tight text-2xl mb-1 @if(in_array($page['controller'], config('base.hero_full_controllers')))xl:text-5xl @else xl:text-3xl @endif">{{ $image['title'] }}</h1>
+                            @if(!empty($image['description']))<div class="pr-2 md:text-lg content  md:white-links">{!! $image['description'] !!}</p>@endif
```
[the remark this answers is not in the document]
Comment (breakdancingcat) [reply resources/views/components/hero.blade.php]: 1. I fixed the space in my last commit
2. White links needs to overwrite the green links in content

**Pull request closed without merging**